docs/transform_it.qmd

```diff
@@ -3,21 +3,25 @@ title: "Transform the output file"
 toc: true
 ---
 
-Once you run SPC to generate a new study area based on your input file text, you will get in `/data/output` folder a `.pb` file. A .pb or [protocol buffer file](https://developers.google.com/protocol-buffers/docs/overview) efficiently encode the the enriched synthetic population [schema](https://github.com/alan-turing-institute/uatk-spc/blob/main/synthpop.proto). You
+Once you run SPC to generate a new study area based on your input file text, you will get in `/data/output` folder a `.pb` file. A .pb or [protocol buffer file](https://developers.google.com/protocol-buffers/docs/overview) efficiently encode the enriched synthetic population [schema](https://github.com/alan-turing-institute/uatk-spc/blob/main/synthpop.proto). You
 can read the "protobuf" (shorthand for a protocol buffer file) in any
 [supported language](https://developers.google.com/protocol-buffers/docs/overview#cross-lang),
 and then extract and transform just the parts of the data you want.
 
-## Python
+## Conda - Python
 
-To work with SPC protobufs in Python, you need two dependencies setup:
+**Optional**: To work with SPC protobufs in Python, you need aditional dependencies dependencies, we have created a conda environment file to assit you to create an independent conda environment to run the scripts included in SPC:
```

Comment:
Also sorry, I think I disagree with doing this. This conda env isn't used to run anything in SPC, or to run code using SPC, so I don't know what the point of providing it is. There are two dependencies listed here, and they should be trivial to add to someone's existing conda env.

Reply:
Ok, np, I’m suggesting as an optional feature.